src/docfx/config/Config.Loader.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal sealed partial class Config
+    {
+        private const string KFiles = "files";
```

Comment:
What does `K` in `KFiles` mean?

Reply:
https://google.github.io/styleguide/cppguide.html#Constant_Names, maybe `kFiles` 😆

Reply:
according to NO.12 in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md use `Files`?